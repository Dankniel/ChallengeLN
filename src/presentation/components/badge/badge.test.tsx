import { beforeEach, describe, expect, test } from 'vitest'
import { render, screen } from '@testing-library/react'
import Badge from './badge'

describe('Badge', () => {

  beforeEach(() => {
    const text = 'Chapita'
    render(<Badge text={text} />)
  })

  test('should render correctly', () => {
    expect(screen.getByText('Chapita')).toBeDefined()
  })
})