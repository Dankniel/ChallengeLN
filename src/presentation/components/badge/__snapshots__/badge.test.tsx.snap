// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Badge > should render correctly 1`] = `
<div>
  <div
    class="badge-container gap-4 align-items-center py-4 px-8 bg-color-black-full border-radius-16 w-fit-content"
  >
    <p
      class="text-color-white font-s-11 lh-16 font-w-400 d-inline-block"
    >
      test
    </p>
  </div>
</div>
`;
